agent-kit/skills/claude-hook-authoring/scripts/validate-hook.sh

```
@@ -0,0 +1,364 @@
+
+if [[ -z "$EVENTS" ]]; then
+  warning "No events defined in hooks configuration"
+fi
+
+TOTAL_HOOKS=0
+
+while IFS= read -r event; do
+  [[ -z "$event" ]] && continue
+
+  # Check if valid event
+  if [[ ! " ${VALID_EVENTS[*]} " =~ " ${event} " ]]; then
```

Shellcheck (0.11.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)